virt/virtual_machines/virt-vtpm-devices.adoc

```diff
@@ -0,0 +1,32 @@
+:_content-type: ASSEMBLY
+[id="virt-vtpm-devices_{context}"]
+= Virtual trusted platform module devices
+include::_attributes/common-attributes.adoc[]
+:context: virt-vtpm-devices
+
+toc::[]
+
+:FeatureName: Virtual trusted platform module devices
+
+A Virtual Trusted Platform Module (vTPM) device protects virtual machines
+(VMs) by storing secrets similarly to a physical Trusted Platform Module (TPM)
+hardware chip.
+
+[NOTE]
+====
+vTPM implementation is non-persistent. vTPM discards stored secrets
+after each shutdown.
+====
+
+You can use a vTPM device with VMs running any operating system. However, if you
+do not enable vTPM, then the VM does not recognize a TPM device, even if the
```

Comment:
How do you enable vTPM? Is that what the "Adding a vTPM device..." module is doing? If not, is there a prereq that needs to be stated?

Reply:
Just add `tpm: {}` to the list of devices, like you would add a soundcard with `sound: {}`.
The device is standalone and doesn't require anything else to function.